backends/advanced/src/advanced_omi_backend/workers/memory_jobs.py

```
@@ -247,6 +247,34 @@ async def process_memory_job(conversation_id: str, *, redis_client=None) -> Dict
+                        user_id=user_id,
+                        conversation_name=conversation_model.title if hasattr(conversation_model, 'title') else None,
+                    )
+                    if kg_result.get("entities", 0) > 0:
+                        logger.info(
+                            f"🔗 Knowledge graph: extracted {kg_result.get('entities', 0)} entities, "
+                            f"{kg_result.get('relationships', 0)} relationships, "
+                            f"{kg_result.get('promises', 0)} promises from {conversation_id}"
+                        )
+                else:
+                    logger.debug("Knowledge graph extraction disabled in config")
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)